pkg/distro/rhel9/partition_tables.go

```diff
@@ -7,10 +7,12 @@ import (
 )
 
 func defaultBasePartitionTables(t *imageType) (disk.PartitionTable, bool) {
-	// RHEL >= 9.3 needs to have a bigger /boot, see RHEL-7999
-	bootSize := uint64(600) * common.MebiByte
+	// RHEL >= 9.3 needs to have a bigger /boot, see RHEL-7999 and COMPOSER-2155
+	bootSize := uint64(1024) * common.MebiByte
```

Comment:
I was playing with switch/case here but I don't think we win anything:
```go
	var bootSize uint64
	switch {
	case common.VersionLessThan(t.arch.distro.osVersion, "9.3") && t.arch.distro.isRHEL():
		bootSize = 500 * common.MebiByte
	case common.VersionLessThan(t.arch.distro.osVersion, "9.4") && t.arch.distro.isRHEL():
		bootSize = 600 * common.MebiByte
	default:
		// RHEL >= 9.3 needs to have a bigger /boot, see RHEL-7999
		bootSize = uint64(600) * common.MebiByte
	}
```
wdyt? probably nested if is fine here

Reply:
Done, thanks!